### DISCUSSION TITLE
error al pushear

### DISCUSSION THREAD

**Discussion by damianperezz** [INGCOM-UNRN-P1/cursada-2023#56] [Preguntas y respuestas] [answered]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141683317/381a75b0-d49d-4de8-8bb1-b7036dac6a80)

Me sale este error cuando quiero pushear. Anteriormente ya habia podido pushear dos ejercicios individualmente, pero quise hacerlo con 3 ejercicios juntos y no me deja.

**Comment by martinvilu** [comment, marked as answer]
Hay cambios en el repositorio remoto que no están en la computadora, mete
,` git pull` antes.

Igual medio raro, estás trabajando en más de una computadora?

**Reply by damianperezz** [reply]
nono, estoy con una sola compu, igualmente me funciono el git pull, muchas gracias!
